test-utils/state-viewer/src/commands.rs

```diff
@@ -50,6 +52,37 @@ pub(crate) fn state(home_dir: &Path, near_config: NearConfig, store: Arc<Store>)
     }
 }
 
+pub(crate) fn dump_contracts(
+    output: &Path,
+    home_dir: &Path,
+    near_config: NearConfig,
+    store: Arc<Store>,
+) {
```

Comment:
In general, we try to improve our error reporting, so perhaps it's better to `-> anyhow::Result<()>` here and output useful error messages when the stuff goes wrong?

Reply:
Agree with the overall sentiment! My gut feeling is that `anyhow` is the right default with high probability. 

* It resists over-engineering quite well, as it is a deliberately "sloppy" type everything else converts to
* it actually makes writing the code easier, as `?` is shorter than `.unwrap()`
* At the same time, when you need to add `.context` to the error, the thing is already there .

That is, I refactored enough of stuff to `anyhow`, that now I tend to just always start with it. 

But this is definitelly unrelated to the PR.

Reply:
@nikurt @matklad This is a good example, on how we can make `neard` cli command errors more readable.

Reply:
At some point in time I realized, that having to read the source code, to figure out what the issue with arguments is. Is a huge waste of time. Usually, I simply want to know, which arguments are wrong, without having to waste time each time reading the code.